v3/gssapi/client.go

```diff
@@ -99,7 +103,7 @@ func (client *Client) DeleteSecContext() error {
 // InitSecContext initiates the establishment of a security context for
 // GSS-API between the client and server.
 // See RFC 4752 section 3.1.
-func (client *Client) InitSecContext(target string, input []byte) ([]byte, bool, error) {
+func (client *Client) InitSecContext(target string, input []byte, APOptions []int) ([]byte, bool, error) {
```

Comment:
We should not break the API.

I suggest adding a new `func (*Client) InitSecContextWithOptions(string, []byte, []int) ([]byte, bool, error)` and then modify the internals of the original `InitSecContext` to simply call the new `InitSecContextWithOptions`, passing in suitable defaults.